### ISSUE TITLE
Unable to open database file.

### ISSUE THREAD

**Issue by gatoravi** [genome/gms#39]
When I try to start a build, I get this error,

![screen shot 2013-11-12 at 8 44 29 pm](https://f.cloud.github.com/assets/764948/1528481/da6b478e-4c0d-11e3-86e2-9824f4352bdd.png)

**Comment by gatoravi** [comment]
The build starts running when I change the permissions of the dir, looks like some permissions issue, any idea where these are set ?

![screen shot 2013-11-13 at 10 21 27 am](https://f.cloud.github.com/assets/764948/1533073/58e5d27e-4c80-11e3-8201-5b71064c2773.png)

**Comment by malachig** [comment]
This could have been related to the permissions error and may have happened at a time when you needed a reboot after a fresh installation.  See related issues: #17 and #24

**Comment by gatoravi** [comment]
this time I ran make as a regular user, the install went through all the way. 
When I tried to start a tst1-tumor-snparray model, I get this error again. There seems to be something wrong with the way  permissions are set for the /opt/gms/GD0T679/sw/ur/lib/UR/DataSource directory. When I do a 'sudo chmod -R 777 /opt/gms/GD0T679/sw/ur/lib/UR/DataSource' (which looks blatantly wrong, I know) the build starts fine.
